Polyhedron/demo/Polyhedron/Scene_polygon_soup_item.cpp

```diff
@@ -683,8 +683,10 @@ Scene_polygon_soup_item::exportAsSurfaceMesh(SMesh *out_surface_mesh)
   CGAL::Polygon_mesh_processing::polygon_soup_to_polygon_mesh< CGAL::Surface_mesh<Point_3> >(
     d->soup->points, d->soup->polygons, *out_surface_mesh);
   std::size_t rv = CGAL::Polygon_mesh_processing::remove_isolated_vertices(*out_surface_mesh);
-  if(rv > 0)
+  if(rv > 0){
     std::cerr << "Ignore isolated vertices: " << rv << std::endl;
+    out_surface_mesh->collect_garbage();
```

Comment:
should be _Ignored_

Reply:
I don't think so. When a plugin ;odifies an item, it calls invalidateOpenGLBuffers() to apply the changes, and collect_garbage() is called in there. It means another fix would have been to call invalidateOpenGLBuffers() instead of collect_garbage(), but as the item has not been drawn  yet I think is more optimized to do like that.